app/src/main/res/values/strings.xml

```diff
@@ -36,63 +36,63 @@
     <!-- Note: Lockdown mode used to be called showcase mode -->
     <string name="manage_repos_pref">Manage repos</string>
     <string name="showcase_mode_pref">Lockdown mode</string>
-    <string name="showcase_mode_desc">Lockdown mode prevent manager to do action on modules</string>
+    <string name="showcase_mode_desc">Lockdown mode prevents the manager from carrying out actions on modules</string>
     <string name="pref_category_settings">Settings</string>
     <string name="pref_category_info">Info</string>
     <string name="show_licenses">Show licenses</string>
     <string name="licenses">Licences</string>
     <string name="show_incompatible_pref">Show incompatible modules</string>
-    <string name="show_incompatible_desc">Show modules that are incompatible with your device based on their metadata</string>
+    <string name="show_incompatible_desc">Show modules unlikely to work on your device based on their metadata</string>
     <string name="magisk_outdated">Magisk is outdated!</string>
     <string name="pref_category_repos">Repos</string>
-    <string name="repo_main_desc">The repository hosting Magisk Modules</string>
-    <string name="repo_main_alt">An alternative to Magisk-Modules-Repo with fewer restrictions.</string>
+    <string name="repo_main_desc">The repository hosting Magisk modules</string>
+    <string name="repo_main_alt">An alternative to the Magisk-Modules-Repo with fewer restrictions.</string>
     <string name="master_delete">Delete the module files?</string>
-    <string name="master_delete_no">Keep files</string>
-    <string name="master_delete_yes">Delete files</string>
-    <string name="master_delete_fail">Failed to delete the module files</string>
+    <string name="master_delete_no">Keep</string>
+    <string name="master_delete_yes">Delete</string>
+    <string name="master_delete_fail">Could not delete the module files</string>
     <string name="theme_pref">Theme</string>
     <string name="theme_mode_pref">Theme mode</string>
-    <string name="module_id_prefix">Module id: </string>
+    <string name="module_id_prefix">Module ID: </string>
     <string name="install_from_storage">Install module from storage</string>
     <string name="invalid_format">The selected module is in an invalid format</string>
```

Comment:
```suggestion
    <string name="invalid_format">The selected module has an invalid format</string>
```
This is minute, but I think if it is the file extention "has" is better, and
if the internal anything is the problem then "is in" is better.